### PR TITLE
[PWGLF] Fix compilation warnings

### PR DESCRIPTION
**Pull request by vkucera**

### CODE REVIEW

**Comment by njacazio** [comment]
Thanks Vit. Some are indeed bugfixes, I will let everyone approve so that they are aware.

**Review by njacazio — DISMISSED** [review]

File: PWGLF/TableProducer/Nuspex/lithium4analysis.cxx
Comment (vkucera) [inline PWGLF/TableProducer/Nuspex/lithium4analysis.cxx:1]: @njacazio Why did you revert these fixes?

**Comment by vkucera** [comment]
Ping

**Comment by vkucera** [comment]
Ping no. 2

**Review by njacazio — APPROVED** [review]

**Pull request merged**